### PR TITLE
Fix lints

### PR DESCRIPTION
**Pull request by CherryKitten**

### CODE REVIEW

**Comment by CherryKitten** [comment]
The last two that are failing right now are things that can't be easily fixed and depend on https://github.com/orgs/famedly/projects/6/views/1?pane=issue&itemId=40709455, so I'd say this is ready for review and will need a follow up later to be fully green

File: roles/systemd-networkd/handlers/main.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 # handlers for ansible-role-systemd-networkd
 
-- name: restart systemd-networkd
+- name: Restart systemd-networkd
```
Comment (jcgruenhage) [inline roles/systemd-networkd/handlers/main.yml:4]: If you rename a handler, you will also need to rename it in the places where it's notified

**Review by jcgruenhage — APPROVED** [review]

**Review by jcgruenhage — CHANGES_REQUESTED** [review]
misclicked, sorry.

**Review by jdreichmann — DISMISSED** [review]
LGTM. the `import_tasks` things would've been ok to just add an ignore imo, but fine

File: roles/systemd-networkd/handlers/main.yml (hunk 1 above)
Comment (CherryKitten) [reply roles/systemd-networkd/handlers/main.yml:4]: ah, thanks for noticing, seems I forgot doing that in this one

**Review by jcgruenhage — APPROVED** [review]

**Pull request merged**